packages/insea-connect-backend/src/main/java/ma/insea/connect/config/SecurityConfig.java

```diff
@@ -34,7 +34,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                         .disable())
                 .headers(headers -> headers.frameOptions(frameOptions -> frameOptions.disable()))
                 .authorizeHttpRequests(
-                        req->req.requestMatchers("/login/**","/register/**")
+                        req->req.requestMatchers("/login/**","/register/**","/*")
```

Comment:
We don't need this one for our api. Just remove it

Reply:
aaah , my bad , i will edit it